### DISCUSSION TITLE
Where can I find examples of how to use line counter with supervison ?

### DISCUSSION THREAD

**Discussion by yeongnamtan** [roboflow/supervision#586] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

I would like to count the number of objects crossing a line.   Where can I find examples for me to test ?

**Comment by SkalskiP** [comment]
Hi, @yeongnamtan! 👋🏻 Thanks a lot for your interest. Let me first convert your question into discussion and move it into Q&A section.

**Comment by SkalskiP** [comment]
@yeongnamtan take a look here:

- YouTube video tutorial: https://www.youtube.com/watch?v=OS5qI9YBkfk
- Blog tutorial: https://blog.roboflow.com/yolov8-tracking-and-counting
- Code example (old Supervision API): https://colab.research.google.com/github/roboflow-ai/notebooks/blob/main/notebooks/how-to-track-and-count-vehicles-with-yolov8.ipynb
- Code example (new Supervision API): https://colab.research.google.com/github/roboflow-ai/notebooks/blob/main/notebooks/how-to-track-and-count-vehicles-with-yolov8-and-supervison.ipynb

**Comment by yeongnamtan** [comment]
thank you @SkalskiP .

Do you have the same for Polygon zone ?
